### PR TITLE
Create truffle project crab corner solution

### PR DESCRIPTION
**Pull request by juliusbuckley**

### CODE REVIEW

File: tutorial/crab-corner-solution/README.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+    ```
+
+4. Compile and migrate the smart contracts. Note inside the development console we don't preface commands with `truffle`.
+    ```javascript
+    compile
+    migrate
+    ```
+
+5. Start the react application by running
+    ```javascript
```
Comment (Lastrellik) [inline tutorial/crab-corner-solution/README.md]: Is this really javascript?

File: tutorial/crab-corner-solution/truffle.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const HDWalletProvider = require('truffle-hdwallet-provider');
+
+const infuraKovanEndpoint = `https://kovan.infura.io/v3/${
+  process.env.INFURA_API_KEY
```
Comment (Lastrellik) [inline tutorial/crab-corner-solution/truffle.js]: Does this require the INFURA_API_KEY environment variable? In my code I've just been hardcoding our endpoint... Which way do you think is better?

File: tutorial/crab-corner-solution/README.md (hunk 1 above)
Comment (juliusbuckley) [reply tutorial/crab-corner-solution/README.md]: it is. it's for code blocks with syntax highlighting 

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

File: tutorial/crab-corner-solution/truffle.js (hunk 2 above)
Comment (juliusbuckley) [reply tutorial/crab-corner-solution/truffle.js]: Yea I generally set all API keys and anything else private as environment variables. I'd recommend switching all those over. Or we can store them in a file that we have set to gitignore.

File: tutorial/crab-corner-solution/README.md (hunk 1 above)
Comment (Lastrellik) [reply tutorial/crab-corner-solution/README.md]: Just the code itself looks like bash

Comment (juliusbuckley) [reply tutorial/crab-corner-solution/README.md]: nice catch. def meant to make that bash

**Pull request merged**